criu/mount.c

```diff
@@ -2501,15 +2510,33 @@ static int do_mount_one(struct mount_info *mi)
 	pr_debug("\tMounting %s @%s (%d)\n", mi->fstype->name, mi->mountpoint, mi->need_plugin);
 
 	if (rst_mnt_is_root(mi)) {
+		int fd;
+		unsigned long flags = MS_BIND | MS_REC;
```

Comment:
It's quiet unrelated to the patch, but maybe we can get rid of MS_REC here?
See more info in https://github.com/checkpoint-restore/criu/issues/754

Reply:
I think this change should be in a separate commit but I'm not sure what would be the result of removing MS_REC. Is it likely to break something?

Reply:
I don't know anybody who actually uses opts.root on mount with children mounts, and it looks weird for me to do so. Just want to attract some attention to these strange MS_REC flag, it looks like a leftover for me from the past dark times =) But yes it's not connected to your patch directly.